modules/ingestor/src/graph/organization.rs

```diff
@@ -83,8 +83,14 @@ impl Graph {
                 let mut entity = organization::ActiveModel::from(found.organization);
                 entity.website = Set(information.website);
                 entity.cpe_key = Set(information.cpe_key);
-                let model = entity.update(&self.connection(&tx)).await?;
-                Ok(OrganizationContext::new(found.graph, model))
+                let model = match entity.update(&self.connection(&tx)).await {
+                    Ok(model) => Ok(model),
+                    Err(e) => {
+                        eprintln!("Failed to update organization: {:?}", e);
```

Comment:
We have some guidelines on how to deal with this: https://github.com/trustification/trustify/blob/main/docs/design/log_tracing.md

Reply:
:facepalm: thanks, and tbh I forgot to remove `eprintln!`

I'll update based on this statement:
`Log messages and tracing should be relevant to operating personell and developers, not end users.` with log::error! macro thanks :+1: